openslides_backend/models/models.py

```diff
@@ -4,7 +4,7 @@
 from openslides_backend.models.base import Model
 from openslides_backend.shared.patterns import Collection
 
-MODELS_YML_CHECKSUM = "83dd0d1d74658e869a797bfebfd0fcc6"
+MODELS_YML_CHECKSUM = "fd53765503552b0562bd71ef2ff437d3"
```

Comment:
Did you generate this from a local models.yml? Please create a PR for it

Reply:
The pull request is on the way.